### PR TITLE
Update the version dropdowns in preparation for 2.4.0 release.

### PR DESCRIPTION
**Pull request by wiwei**
Updating the dropdown to where it will end up being, rather than just where it is now.

### CODE REVIEW

**Comment by davidkline-ms** [comment]
One note: this needs to be done in the mrtk_develooment branch

File: web/version.js [hunk 1]
```diff
@@ -2,7 +2,15 @@ function createDropdown()
...
 	var defaultTitle = "mrtk_development"; // title in the dropdown for the root version of the docs
-	var versionArray = ["releases/2.0.0", "releases/2.1.0", "releases/2.2.0", "releases/2.3.0", "prerelease/2.4.0_stabilization"]; // list of all versions in the version folder
+
+	// list of all versions in the version folder
+	var versionArray = [
+		"releases/2.0.0",
+		"releases/2.1.0",
+		"releases/2.2.0",
+		"releases/2.3.0",
+		"releases/2.4.0"
```
Comment (keveleigh) [inline web/version.js:12]: On a similar note, should we update our in-code documentation references to point to the actual release?

https://github.com/microsoft/MixedRealityToolkit-Unity/blob/31f1a04118d3a35a1c7855d0a01034bfa7a0cb77/Assets/MRTK/Core/Utilities/StandardShader/BaseMeshOutline.cs#L11

**Comment by wiwei** [comment]
> One note: this needs to be done in the mrtk_develooment branch

Yep, this will happen when we push 2.4.0_stabilization back down to mrtk_development

File: web/version.js (hunk 1 above)
Comment (davidkline-ms) [reply web/version.js:12]: That would likely require a second PR (to dev) that restores the in-code links.

Comment (keveleigh) [reply web/version.js:12]: Yep, similar to what we did with NuGet in the past: https://github.com/microsoft/MixedRealityToolkit-Unity/pull/7278

**Comment by CDiaz-MS** [comment]
This branch does not exist yet, will this change cause issues now?

**Comment by thalbern** [comment]
added do not merge label so we don't accidentally merge it until the release is ready :)

**Pull request merged**